### ISSUE TITLE
Exception thrown when a project is opened: java.lang.NullPointerException

### ISSUE THREAD

**Issue by kherge** [hsz/idea-gitignore#130]
I seen this exception consistently for the first project I open after a cold start. If I close the project and open a different one, the exception is not thrown again until I restart PhpStorm and open a project.

```
null
java.lang.NullPointerException
	at mobi.hsz.idea.gitignore.util.Utils$2.compare(Utils.java:436)
	at mobi.hsz.idea.gitignore.util.Utils$2.compare(Utils.java:433)
	at java.util.TimSort.binarySort(TimSort.java:292)
	at java.util.TimSort.sort(TimSort.java:217)
	at java.util.Arrays.sort(Arrays.java:1512)
	at java.util.ArrayList.sort(ArrayList.java:1454)
	at java.util.Collections.sort(Collections.java:175)
	at com.intellij.util.containers.ContainerUtil.sort(ContainerUtil.java:1521)
	at mobi.hsz.idea.gitignore.util.Utils.ignoreFilesSort(Utils.java:433)
	at mobi.hsz.idea.gitignore.IgnoreManager$5.run(IgnoreManager.java:365)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:318)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by hsz** [comment]
@kherge Thank you for reporting.

Fix will be released with the next version.

**Comment by hsz** [comment]
@kherge  Fix has been already released with version v1.1.3 and is available for download.
